GETTING-STARTED.md

```
@@ -42,35 +49,30 @@ bin/act --secret-file .env
 
 ## Deploying
 
-Important: For now the target infrastructure is not available. This is why we currently only do manual deployments. If
-you really want to deploy, create an issue to ask for permissions.
+**Important**: For now the target infrastructure is not fully available. Manual deployments are currently in progress. If you need to deploy, create an issue to request permissions.
 
-Build and push images to Azure Container Registry
+### Build and Push Images to Azure Container Registry
 
+First, authenticate with the registry:
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)